modules/administration/pages/tshoot-firewalls.adoc

```diff
@@ -0,0 +1,47 @@
+[[troubleshooting-firewalls]]
+= Troubleshooting Firewalls
+
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+
+
+Cause: User firewall is set to block outgoing traffic by dropping the packet request. During sync with SCC, SUMA waits for an answer on each URL until it times out, eventually causing the entire refresh of the product list to timeout. Applies only to third party (non-SUSE) products, as sync with SCC needs to access locations other than SCC to verify if the if the download location is valid.
+
+Consequence: The sync to SCC fails, and the third party products are not shown in the product list.
+
+Fix: Configure the firewall to reject requests from SUMA instead of drop (preferred), or configure a firewall on the server (if no ability to change firewall settings)
+
+Result: Sync to SCC will either be able to reach the URLs required, or will have request rejected so that the request fails rather than times out.
+////
+
+If you are using a firewall that blocks outgoing traffic, it will either ``REJECT`` or ``DROP`` network requests.
+If it is set to ``DROP`` then you might find that synchronizing with the {scc} times out when you are using non-{suse} clients, including {centos}, {ubuntu}, and {oracle}.
+
+This occurs because the synchronization process for non-{suse} clients needs to access the third-party repositories that provide packages for these clients, and not just the {scc}.
+When the {productname} Server attempts to reach these repositories to check that they are valid, the firewall drops the requests, and the synchronization continues to wait for the response until it times out.
+
+If this occurs, you will notice that the synchronization will take a long time before it fails, and that your non-{suse} products are not shown in the product list.
+
+You can fix this problem in several different ways.
+
+The simplest method is to configure your firewall to allow access to the URLs required by the non-{suse} repositories you are using.
+This allows the synchronization process to reach the URLs and complete successfully.
+
+If allowing external traffic is not possible, configure your firewall to ``REJECT`` requests from {productname} instead of ``DROP``.
+This rejects requests to third-party URLs, so that the synchronization fails rather than times out.
```

Comment:
```suggestion
This rejects requests to third-party URLs, so that the synchronization fails early rather than times out, and the products are not shown in the list.
```

Reply:
Hallo Uyuni-Team,
what about Debian Client System

modules/administration/pages/tshoot-firewalls.adoc 
>> when you are using non-{suse} clients, including {centos}, {ubuntu}, and {oracle}.

Could you please add {debian) also.

Thanks in advance.
Best wishes.
Shirocco88

Reply:
Hi Shirocco88, two points on this: Firstly, I've removed that line now for other reasons (see earlier comments), and secondly, this is just a list of example third-party clients. The purpose of adding a couple of examples is to make it clearer to the reader what we mean by "third-party" or "non-SUSE" clients. For the purposes of example, I picked the most common third-party clients, it was not intended to be an exhaustive list.